apis/node/src/local.ts

```diff
@@ -22,7 +25,51 @@ function processError(res: Response, err: any) {
   res.write(`${err}`);
 }
 
-app.post("/stream/completion", async (req, res, next) => {
+function isTestingMode() {
+  return Env.localCachePath.length > 0;
+}
+
+let _testingCache: Record<string, string> | null = null;
+async function loadTestingCache() {
+  if (_testingCache === null) {
+    const contents = await fs.promises.readFile(Env.localCachePath, {
+      encoding: "utf-8",
+    });
+    _testingCache = JSON.parse(contents) as Record<string, string>;
+  }
+  return _testingCache;
+}
```

Comment:
I think this would probably be much cleaner implemented in the main braintrust repo as a separate server (using node) that happens to use these cache functions, vs. littering the main implementation.

Of note, any additional logic we place here bloats the bundle that gets deployed to lambda